### ISSUE TITLE
Could use help in moving to beta.1

### ISSUE THREAD

**Issue by born2net** [born2net/ng2Boilerplate#3]
Tried to move the project to beta.1 and got some errors with dependency injection and from a quick look it seems to be bugs related to ng2 latest beta.1 .... BUT I didn't dive deep yet!

If anyone has some time and can assist would be appreciated..

tx

Sean

**Comment by born2net** [comment]
fixed beta.1 and added webpack support

**Comment by dancancro** [comment]
This project is great, and I don't want to create an issue to say that. It's a shame that I didn't come across this sooner. Thinking that there was no full-on example app out there in the public view, I've been working on [this one](https://github.com/dancancro/great-big-angular2-example) of my own.

There really needs to be a better way to get the good ones more publicity.

My efforts towards that goal is [this database of projects and their selling points](https://docs.google.com/spreadsheets/d/1nMv8TqUx3gUoC3M6BRPB4E4FMTSYGT_OLERguXGjePc/edit#gid=1404564369) along with [this user interface](http://techtradeoffs.info) to that data.

I fill it in from time to time and my interest ebbs and flows but if a project owner wants to fill in the info about his project, the help is always welcome. Let me know if you'd like to add this one to it.

**Comment by born2net** [comment]
absolutely...
I also recommend: https://github.com/born2net/ng-skeleton
and I agree, we need a central repo of all the cool ng stuff!

**Comment by dancancro** [comment]
Great. I added two new columns, P & Q, for these to the spreadsheet. Don't worry about the first few rows with the percentages. That was an attempt I made to quantify things, but they use weights which are subjective.

Just enter in Yes as comments to appropriate cells for benefits the projects deliver directly. I'll take care of entering the project dependencies. I'm sure these projects depend on technologies that aren't in the spreadsheet yet. Let me know of the ones you think are most important and I'll add columns for those too.
